pkg/cmd/pr/diff/diff.go

```diff
@@ -134,3 +142,10 @@ func isRemovalLine(dl string) bool {
 func validColorFlag(c string) bool {
 	return c == "auto" || c == "always" || c == "never"
 }
+
+var runPager = func(pager string, diff io.Reader, out io.Writer) error {
+	pagerCmd := exec.Command(pager)
```

Comment:
A pager command read from PAGER might include arguments such as flags (e.g. `PAGER="less -R"`), so I think we should `shlex.Split()` that string and forward all the arguments to `exec.Command(...)`

Otherwise this code is going to try to find an executable literally named `less -R` and fail.

Reply:
done!